packages/apps/registry-backend/src/lib/express/app/routes.ts

```diff
@@ -20,6 +20,23 @@ function generateRandomAppId(): number {
   return Math.floor(Math.random() * (10_000_000_000 - 1000)) + 1000;
 }
 
+async function addDelegateesToPaymentDB(delegateeAddresses: string[]): Promise<void> {
+  // hit the relayer server to add the delegatee addresses
+  const headers = {
+    'api-key': process.env.LIT_RELAYER_API_KEY!,
```

Comment:
I know using the ! is not correct, but how can I set it up so that like, the server won't even start if these env vars are not present?  and then I don't need to check for their presence every time I use them?

Reply:
Exactly, `createEnv` in that file does exactly what you want @glitch003 . It can also apply validations such as format, optional, default value and make them the right type

Then just read the env var from that file, [for example here](https://github.com/LIT-Protocol/Vincent/blob/main/packages/apps/registry-backend/src/lib/express/index.ts#L17)